File: Python/swapper.py
def moveElementToEnd(array, toMove):
    # Write your code here.
	i=0
	swapper = len(array)-1
	while i<swapper:
		while i<swapper and array[swapper]==toMove:
			swapper -=1
		if array[i]==toMove:
			array[i] , array[swapper] = array[swapper], array[i]
		i +=1	
    return array

File: C++/swapper.cpp
#include <vector>

using namespace std;

vector<int> moveElementToEnd(vector<int> array, int toMove) {
  // Write your code here.
	int swapper=array.size()-1;
	int i=0;
	while(i<swapper){
	
		while(i<swapper&&array[swapper]==toMove){
			swapper--;
		}
			if(array[i]==toMove)
				swap(array[i],array[swapper]);
			i++;
		
	}

	
  return {array};
}
